tests/basis_test.py

```diff
@@ -3,328 +3,274 @@
 import numpy
 from scipy import special
```

Comment:
Try to avoid using from imports, as they pollute the namespace
```suggestion
import scipy.special
```
Update as needed below